README.md

```diff
@@ -30,19 +30,17 @@ You don't need to have a PHP environment with composer installed.
 ### Follow these steps
 
 1. Initialize the demo project using the `pimcore/pimcore` image
-    ``docker run --rm -v `pwd`:/var/www/html pimcore/pimcore:PHP8.0-fpm composer create-project pimcore/demo my-project``
-1. Fix permissions of the created directory (we are already working on a fix to make this redundant)
-    `sudo chown -R 1000 my-project/`
+    ``docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:PHP8.0-fpm composer create-project pimcore/demo my-project``
 1. Go to your new project
     `cd my-project/`
-1. Part of the new project is a docker compose file. Use it to start the needed services
-    `docker-compose up -d`
+1. Part of the new project is a docker compose file
+    * Run `` echo `id -u`:`id -g` `` to retrieve your local user and group id
+    * Open the `docker-compose.yml` file in an editor, uncomment all the `user: '1000:1000'` lines and update the ids if necessary
+    * Start the needed services with `docker-compose up -d`
 1. Install pimcore and initialize the DB
-    `docker-compose exec php-fpm vendor/bin/pimcore-install --mysql-host-socket=db --mysql-username=pimcore --mysql-password=pimcore --mysql-database=pimcore`
+    ``docker-compose exec -u `id -u`:`id -g` php-fpm vendor/bin/pimcore-install --mysql-host-socket=db --mysql-username=pimcore --mysql-password=pimcore --mysql-database=pimcore``
```

Comment:
Is it really necessary to add `-u ...` after it was already set in the previous step?

Reply:
Good point, that can be skipped. Updated my PR